packages/suite-desktop/src-electron/modules/shortcuts.ts

```diff
@@ -9,6 +9,11 @@ const init = ({ mainWindow, src }: Dependencies) => {
         mainWindow.webContents.openDevTools();
     });
 
+    electronLocalshortcut.register(mainWindow, 'CommandOrControl+Shift+I', () => {
```

Comment:
why do we need 2 different shortcuts for that? `CommandOrControl+Alt+I` (above) and `'CommandOrControl+Shift+I`

Reply:
to keep code clean I would suggest iterating an array and keep both (or even more?) shortcuts

Reply:
fixed in 74f2142